Eastern-Research-Group/expert-query
# Feature/301 fix cypress tests

## Related Issues:
* [EQ-301](https://jira.epa.gov/browse/EQ-301)

## Main Changes:
* Updated the cypress tests to account for recent changes in the UI, and changes in column values returned by the server since adding type parsers.
* Fixed a bug in the server that cropped up when extracting a function used in two endpoints.

## Steps To Test:
1. Ensure that the server is pointed at the development database.
2. Run the Cypress tests, and verify they all pass.

## TODO:
- [x] I merged https://github.com/Eastern-Research-Group/expert-query/pull/124/commits/deac3be7ecd3507acf6ac4ed36e2fd3c05093bae into this branch, since it looked like it would cause merge conflicts, so PR #124 should be reviewed first.
